src/app/modules/login/pages/login/login.page.ts

```diff
@@ -1,15 +1,37 @@
 import { Component, OnInit } from "@angular/core";
+import { Validators, FormBuilder } from "@angular/forms";
+import { AuthService } from "@core/authentication/authentication.service";
 
 @Component({
-  selector: "app-login-page",
-  templateUrl: "./login.page.html",
-  styleUrls: ["./login.page.scss"]
+    selector: "app-login-page",
+    templateUrl: "./login.page.html",
+    styleUrls: ["./login.page.scss"],
 })
 export class LoginPageComponent implements OnInit {
+    loginForm = this.fb.group({
+        email: ["", [Validators.required, Validators.email]],
+        password: ["", Validators.required],
+    });
 
-  constructor() { }
+    constructor(
+        private fb: FormBuilder,
+        private authService: AuthService,
+    ) {}
 
-  ngOnInit(): void {
-  }
+    ngOnInit(): void {
+    }
 
+    onSubmitLogin() {
+        if (this.loginForm.valid) {
+            this.authService.login(this.email, this.password);
+        }
+    }
+
+    get email(): string {
```

Comment:
getters should be placed above the constructor.

Okay for now, we can change it up later